common.sh

```diff
@@ -71,99 +99,165 @@ log()
         fi
     fi
 
-    echo "$SEVERITY $COMPONENT: $1"
+    echo "$SEVERITY $COMPONENT: $1" >&2
```

Comment:
Are we sure all logs should go to stderr? What goes to stdout? I'm not disagreeing, just curious; this will need to be reflected in our Red Hat internal test API.